pandas/core/series.py

```diff
@@ -2944,7 +2944,7 @@ def _try_kind_sort(arr):
             sortedIdx[n:] = idx[good][argsorted]
             sortedIdx[:n] = idx[bad]
         else:
-            raise ValueError("invalid na_position: {!r}".format(na_position))
+            raise ValueError(f"invalid na_position: {na_position}")
```

Comment:
```suggestion
            raise ValueError(f"invalid na_position: {repr(na_position)}")
```

OK to replace the previous `!r` specifier with a `repr` call here

Reply:
don't need repr on a boolean